Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -291,12 +291,12 @@ To learn more about the final data format created by Edge Bridge, please refer t
 #### Trigger rule-based `trackAction` events <!-- omit in toc -->
 Rule-based `trackAction`/`trackState` events are also converted to Edge events by the Edge Bridge extension. Select the **Trigger Consequence** button (**1**) to initiate a rule that generates a `trackAction` event.
 
-<img src="../assets/edge-bridge-tutorial/assurance-validation/ios-app-trigger-rule-button.png" alt="Simulator tracking buttons" width="400"/>
+<img src="../assets/edge-bridge-tutorial/assurance-validation/ios-app-trigger-consequence-button.png" alt="Simulator tracking buttons" width="400"/>
 
 > [!TIP]
-> If no `trackAction` event is dispatched upon selecting the **Trigger Consequence** button, verify that your mobile property is [set up with the correct rule](#2-configure-a-rule-to-forward-personal-identifiable-information-pii-events-to-edge-network).
+> If no `trackAction` event is dispatched upon selecting the **Trigger Consequence** button, verify that your mobile property is [set up with the correct rule](#2-configure-a-rule-to-forward-personally-identifiable-information-pii-events-to-edge-network).
```

Comment:
I click the link but can't tell which link it supposes to go to.

Reply:
That works, so the user will know which section to look for from the Adobe Experience Platform setup header.